plugins/copy-to-clipboard/prism-copy-to-clipboard.js

```diff
@@ -35,13 +35,39 @@
 		head.appendChild(script);
 	}
 
+	function getSettings(start, defaults) {
+		var prefix = 'data-prismjs-';
+		var settings = defaults;
+		for (var key in settings) {
+			var attr = prefix + key;
+			var element = start;
+			while (element && !element.hasAttribute(attr)) {
+				element = element.parentElement;
+			}
+			if (element !== null) {
+				var value = element.getAttribute(attr);
+				if (typeof value !== 'undefined') {
```

Comment:
[`getAttribute`](https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute) will always return `null` or a string. It cannot return `undefined`.

The check is unnecessary anyway because we know that `element.hasAttribute(attr)` is `true` which means that `element.getAttribute(attr)` will return a string.

Reply:
Thanks! It should’ve been `null`. I’ll remove this line.